tests/util/kartutil.go

```diff
@@ -0,0 +1,273 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package util
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"math/rand"
+	"os/exec"
+	"regexp"
+	"strings"
+	"time"
+
+	kins "github.com/kubearmor/kubearmor-client/install"
+	kcli "github.com/kubearmor/kubearmor-client/k8s"
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/remotecommand"
+	"k8s.io/kubectl/pkg/scheme"
+)
+
+var k8sClient *kcli.Client
+var stopChan chan struct{}
+
+func isK8sEnv() bool {
+	if k8sClient != nil {
+		return true
+	}
+	cli, err := kcli.ConnectK8sClient()
+	if err != nil {
+		return false
+	}
+	k8sClient = cli
+	return true
+}
+
+func getOptions() kins.Options {
+	return kins.Options{
+		"kube-system",
+		"kubearmor/kubearmor:stable",
```

Comment:
We would need to use locally built test images to test things right 🤔

Reply:
We are currently executing `make run` and using the test framework against that. But yes, we will need a way to test with local images.